### ISSUE TITLE
VQA finetune: test annotation file test-dev2015_qla_mrcnn.json and test2015_qla_mrcnn.json are missing

### ISSUE THREAD

**Issue by yangapku** [microsoft/Oscar#74]
Hi, thank you for releasing the great work! I am working on the VQA task. May I ask where can I find the annotation file `test2015_qla_mrcnn.json` and `test-dev2015_qla_mrcnn.json` to make inference on the test set? It seems these files do not exist in the download path `https://biglmdiag.blob.core.windows.net/vinvl/datasets/vqa/`. Thank you very much.

**Comment by yangapku** [comment]
Meanwhile, the file `train+val2014_qla_mrcnn.json` to finetune on both the train and valid set is also missing.

**Comment by maxinge8698** [comment]
> Hi, thank you for releasing the great work! I am working on the VQA task. May I ask where can I find the annotation file `test2015_qla_mrcnn.json` and `test-dev2015_qla_mrcnn.json` to make inference on the test set? It seems these files do not exist in the download path `https://biglmdiag.blob.core.windows.net/vinvl/datasets/vqa/`. Thank you very much.

Hello, may I ask if you have found these files? I have the same problem as you did then. Your help is badly needed.

**Comment by yangapku** [comment]
@maxinge8698 No I haven't found these files. Now I have managed to implement my own model, which is named OFA and have achieved a better accuracy than VinVL (test-std 80.02) as well as on many other tasks. Maybe you can also turn to try our OFA https://github.com/OFA-Sys/OFA.

**Comment by maxinge8698** [comment]
> @maxinge8698 No I haven't found these files. Now I have managed to implement my own model, which is named OFA and have achieved a better accuracy than VinVL (test-std 80.02) as well as on many other tasks. Maybe you can also turn to try our OFA https://github.com/OFA-Sys/OFA.



> @maxinge8698 No I haven't found these files. Now I have managed to implement my own model, which is named OFA and have achieved a better accuracy than VinVL (test-std 80.02) as well as on many other tasks. Maybe you can also turn to try our OFA https://github.com/OFA-Sys/OFA.

Thank you very much for your help. Your new work looks very good.
